js/scenes/Home/Home.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  View,
+  Text
+}from 'react-native';
+
+import { styles } from './styles';
+
+import HomeTab from '../../components/HomeNavigationTab';
+
+const Home = () => {
+
+  return (
+    <View style={ styles.container }>
+      <View style={ styles.ownerSection }>
+         <Text style={ styles.homeHeader }>
+          Pet Owner
+        </Text>
+
+        <View style={ styles.tabContainer }>
+          <HomeTab />
+
+          <HomeTab />
+        </View>
+      </View>
+
+      <View style={ styles.volunteerSection }>
+        <Text style={ styles.homeHeader }>
+          Volunteer
+        </Text>
+
+        <View style={ styles.tabContainer }>
+          <HomeTab />
```

Comment:
Each <HomeTab /> represents one of the rectangular navigation buttons on the front page. We've modularized the tab component so we can reuse it and just feed in the required information as props (which we'll get to later).